pimd/pim_instance.h

```diff
@@ -178,8 +178,9 @@ struct pim_instance {
 	struct list *ssmpingd_list;
 	struct in_addr ssmpingd_group_addr;
 
-	unsigned int igmp_group_count;
-	unsigned int igmp_watermark_limit;
+	/*IGMP or MLD Group count*/
+	unsigned int group_count;
+	unsigned int watermark_limit;
```

Comment:
Please put `gm_` in front of these 2 variables, since they are in `pim_instance` it is not clear otherwise if they refer to IGMP or PIM.

(the other changes are OK because they are in `struct igmp_xxx`, but this is `struct pim_instance` which is mixed with a lot of PIM data)

Reply:
Fixed